code/modules/mob/living/carbon/human/life.dm

```diff
@@ -113,16 +113,19 @@
 
 ///FIRE CODE
 /mob/living/carbon/human/handle_fire()
-	if(!last_fire_update)
-		last_fire_update = fire_stacks
```

Comment:
this was being updated every tick of life() since ..() wasn't being checked to see if the mob was put out or not.